objcgen/EqualsHelper.cs

```diff
@@ -7,7 +7,7 @@ public class EqualsHelper : MethodHelper{
 		public EqualsHelper (SourceWriter headers, SourceWriter implementation) : base (headers, implementation)
 		{
 			MonoSignature = "Equals(object)";
-			ObjCSignature = "isEqual:(id _Nullable)other";
+			ObjCSignature = "areEqual:(id _Nullable)other";
```

Comment:
that's not correct, `isEqual:` is the correct signature for one argument (not operator) and is also what `NSObject` protocol defines (and what we need to conform)